.github/workflows/go.yml

```
@@ -0,0 +1,32 @@
+name: Go
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    name: "Build"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: ["1.18"]
+    steps:
+      - name: Updating ...
+        run: sudo apt-get -qq update
+
+      - uses: actions/checkout@v3
+
+      - name: Setup go
+        uses: actions/setup-go@v4
```

This line throws a warning. I'm not sure if this can be fixed by specifying the path to `go.sum`
```suggestion
        uses: actions/setup-go@v4
        with:
            cache-dependency-path: ./go.sum
```
 or disabling caching: 

```suggestion
        uses: actions/setup-go@v4
        with:
            cache: false
```

---

I read over [the section on caching](https://github.com/actions/setup-go#caching-dependency-files-and-build-outputs) in the setup-go action. From what I understand from the section, the cache should be set up automatically. Using `cache-dependency-path` should not be necessary, as `go.sum` exists in the current directory. The action searches for `go.sum` automatically.